### PR TITLE
Updated explanation when maximum timeout will be used

### PR DESCRIPTION
**Pull request by nsuma8989**

### CODE REVIEW

**Comment by mbelton-buildkite** [comment]
Hey @nsuma8989, can you tell me a bit more about where this came from? Was there a support case relating to this?

**Comment by mcncl** [comment]
@mbelton-buildkite yep, we've had a couple of folks ask for clarity on this this week

**Comment by nsuma8989** [comment]
@mbelton-buildkite As @mcncl mentioned we got few customers reaching out for clarification on this so that was the reason behind this PR. Please let me know how we want to proceed on this

File: pages/pipelines/command_step.md [hunk 1]
```diff
@@ -214,7 +214,7 @@ To prevent jobs from consuming too many job minutes or running forever, you can
 
 Specific timeouts take precedence over more general ones — a step level timeout takes precedence over a pipeline timeout, which in turn takes precedence over an organization default.
 
-Maximum timeouts are applied to any command step without a timeout or with a timeout greater than the maximum timeout.
+Maximum timeouts are applied to any command step without a timeout, default timeout not set in pipeline settings or with a timeout greater than the maximum timeout.
```
Comment (mbelton-buildkite) [inline pages/pipelines/command_step.md:217]: Style: Use a serial comma.

```suggestion
Maximum timeouts are applied to any command step without a timeout, default timeout not set in pipeline settings, or with a timeout greater than the maximum timeout.
```

Comment (mbelton-buildkite) [reply pages/pipelines/command_step.md:217]: Actually, I think the issue is how hard the sentence is to parse. How about splitting it into a bulleted list? Thinking:

```suggestion
Maximum timeouts are applied to command steps in the following situations:

- No timeout attribute is set on the step.
- No default timeout is set in the pipeline settings.
- When the timeout set is greater than the maximum timeout.
```

Comment (nsuma8989) [reply pages/pipelines/command_step.md]: @mbelton-buildkite I like this bullet list approach as it is easy to read and understand. I updated the PR with this approach

File: pages/pipelines/command_step.md [hunk 2]
```diff
@@ -214,7 +214,10 @@ To prevent jobs from consuming too many job minutes or running forever, you can
 
 Specific timeouts take precedence over more general ones — a step level timeout takes precedence over a pipeline timeout, which in turn takes precedence over an organization default.
 
-Maximum timeouts are applied to any command step without a timeout or with a timeout greater than the maximum timeout.
+Maximum timeouts are applied to command steps in the following situations:
```
Comment (mbelton-buildkite) [inline pages/pipelines/command_step.md:217]: Style: You need to include a blank line before a list for it to render correctly. At the moment, it looks like:

Screenshot 2023-04-20 at 8 20 54 am

```suggestion
Maximum timeouts are applied to command steps in the following situations:

```

File: pages/pipelines/command_step.md [hunk 3]
```diff
@@ -214,7 +214,10 @@ To prevent jobs from consuming too many job minutes or running forever, you can
...
+- No timeout attribute is set on the step.
+- No default timeout is set in the pipeline settings.
+- When the timeout set is greater than the maximum timeout
```
Comment (mbelton-buildkite) [inline pages/pipelines/command_step.md:220]: Typo: Missing period.

```suggestion
- When the timeout set is greater than the maximum timeout.
```

**Review by mbelton-buildkite — APPROVED** [review]
Two small nitpicks to fix, then all good :)

**Comment by nsuma8989** [comment]
> Two small issues to fix, then all good :)

Thank you. I applied those changes.

**Comment by mbelton-buildkite** [comment]
Looks good! Merge when you're ready :)

**Pull request merged**